lib/linguist/languages.yml

```diff
@@ -5022,6 +5022,31 @@ World of Warcraft Addon Data:
   tm_scope: source.toc
   ace_mode: text
   language_id: 396
+X BitMap:
+  type: data
+  group: C
+  aliases:
+  - xbm
+  extensions:
+  - ".xbm"
+  ace_mode: c_cpp
+  tm_scope: source.c
+  codemirror_mode: clike
+  codemirror_mime_type: text/x-csrc
+  language_id: 782911107
+X PixMap:
```

Comment:
Aaaah, man. This is a PITA for me. I need to relearn Rails migrations if we do this. If I don't put in a migration, we end up with the same problems we had with the Pelr6 -> Perl 6 change and the DB and search gods frown upon me.

Reply:
Well, erh, should we then start thoroughly reviewing the hell out of every single grammar-name to make any other potential future change happen in one migration? 😅

I didn't want to rename the entry, as I *am* conscious of how much of a pain it is database-wise. 😞 Unfortunately, I can't change fact. :( That's a fact.

Reply:
If the KiCad incident last year proved anything, it's that inaccuracies in `language.yml` are better dealt with much sooner than a hell of a lot later. 😥